elixir/apps/api/lib/api/client/channel.ex

```diff
@@ -766,16 +770,24 @@ defmodule API.Client.Channel do
   end
 
   def map_or_drop_compatible_resource(resource, client_or_gateway_version) do
-    if Version.match?(client_or_gateway_version, ">= 1.2.0") do
-      {:cont, resource}
-    else
-      resource.address
-      |> String.codepoints()
-      |> Resources.map_resource_address()
-      |> case do
-        {:cont, address} -> {:cont, %{resource | address: address}}
-        :drop -> :drop
-      end
+    cond do
+      resource.type == :internet and Version.match?(client_or_gateway_version, ">= 1.3.0") ->
```

Comment:
I think we'll roll this out with the other changes for client 1.2.0 release.

```suggestion
      resource.type == :internet and Version.match?(client_or_gateway_version, ">= 1.2.0") ->
```

Reply:
We should not do that. 1.2 is for the wildcard. Better we bump the version twice because then we will have a way to develop internet resources and deploy without breaking everything.

Reply:
K, in that case I think we can release new clients today-ish and deploy prod right away, right?

Reply:
cc @conectado to confirm.

Reply:
I just realized that after merging this PR #6325if we send the internet resource to clients currently in main it might break some clients. Since now we send the resource to the clients, some of the clients can't support the deserialization of the internet resource.

Will have a PR asap to fix this.

Reply:
Fixed in #6381